src/InputWithLabel.js

```diff
@@ -0,0 +1,27 @@
+import React, {useRef, useEffect} from 'react'
+
+function InputWithLabel({ children, value, onChange}) {
+    const inputRef = useRef(null)
+
+    useEffect(() => {
+         // Accessing the input element and focusing it
+        inputRef.current.focus();
+    }, []);
```

Comment:
This is very unintuitive but having an empty array for the dependency array argument triggers the effect only on the initial render. In this case we want to trigger the effect on *every* render. To support that we want no dependency array argument at all (leaving it `undefined`)

```suggestion
    });
```

Reply:
after you make this change you should see the input become re-focused after every time you click the "Add" or "Remove" buttons

Reply:
Thanks a lot! I'll make the change and merge afterwards